java/ql/src/semmle/code/java/security/UnsafeDeserialization.qll

```diff
@@ -109,6 +176,119 @@ class SafeKryo extends DataFlow2::Configuration {
   }
 }
 
+private class EnableJacksonDefaultTypingConfig extends DataFlow2::Configuration {
+  EnableJacksonDefaultTypingConfig() { this = "EnableJacksonDefaultTypingConfig" }
+
+  override predicate isSource(DataFlow::Node src) {
+    any(EnableJacksonDefaultTyping ma).getQualifier() = src.asExpr()
+  }
+
+  override predicate isSink(DataFlow::Node sink) { sink instanceof ObjectMapperReadSink }
+}
+
+private class SafeObjectMapperConfig extends DataFlow2::Configuration {
```

Comment:
```suggestion
/** 
 * Tracks flow from qualifiers to `[set]PolymorphicTypeValidator(...)` calls to a subsequent `readValue` or similar call, including across builder method calls.
 *  
 * Such a `readValue` call is safe because validation will likely prevent instantiating unexpected types.
 */
private class SafeObjectMapperConfig extends DataFlow2::Configuration {
```